### ISSUE TITLE
video fails to be saved if the filename contains spaces

### ISSUE THREAD

**Issue by efstathiosntonas** [maitrungduc1410/react-native-video-trim#74]
Hi, thank you for this lib.

This is a rare case probably but I tried to trim a video that has spaces in it's filename and it failed on iOS, haven't tested Android yet. Normally videos captured with camera won't have spaces but I've just sent a video from my desktop using AirDrop to my iPhone 15 Pro running iOS 18 and it failed.

This is the event log:

```log
 LOG  {"name": "onShow"}
 LOG  {"duration": 40247.9, "name": "onLoad"}
 LOG  {"name": "onStartTrimming"}
 LOG  {"command": "-ss 0.0ms -to 30000.0ms -i file:///var/mobile/Containers/Data/Application/85F099A5-5DCB-4794-B055-53E15AD942D0/tmp/Pixel%2040%20second%20APP%20video.mp4 -c copy -metadata creation_time=2024-10-08T07:37:34.543000+0000 file:///var/mobile/Containers/Data/Application/85F099A5-5DCB-4794-B055-53E15AD942D0/Documents/trimmedVideo_1728373054.mp4", "name": "onLog"}
 LOG  {"level": 32, "message": "ffmpeg version n6.0", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": " Copyright (c) 2000-2023 the FFmpeg developers", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  built with Apple clang version 14.0.3 (clang-1403.0.22.14.1)
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  configuration: --cross-prefix=arm64-ios-darwin- --sysroot=/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS16.4.sdk --prefix=/Users/sue/Projects/arthenica/ffmpeg-kit/prebuilt/apple-ios-arm64/ffmpeg --pkg-config=/opt/homebrew/bin/pkg-config --enable-version3 --arch=aarch64 --cpu=armv8 --target-os=darwin --enable-neon --enable-asm --ar=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar --cc=clang --cxx=clang++ --as='/Users/sue/Projects/arthenica/ffmpeg-kit/.tmp/gas-preprocessor.pl -arch aarch64 -- clang -arch arm64 -target arm64-apple-ios12.1 -march=armv8-a+crc+crypto -mcpu=generic -DFFMPEG_KIT_ARM64 -Wno-unused-function -Wno-deprecated-declarations -fstrict-aliasing -DIOS -DFFMPEG_KIT_BUILD_DATE=20230903 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS16.4.sdk -Oz -miphoneos-version-min=12.1 -I/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS16.4.sdk/usr/include' --ranlib=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib --strip=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/strip --nm=/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/nm --extra-ldflags='-miphoneos-version-min=12.1' --disable-autodetect --enable-cross-compile --enable-pic --enable-inline-asm --enable-optimizations --enable-swscale --enable-shared --disable-static --install-name-dir='@rpath' --enable-pthreads --disable-v4l2-m2m --disable-outdev=v4l2 --disable-outdev=fbdev --disable-indev=v4l2 --disable-indev=fbdev --enable-small --disable-xmm-clobber-test --disable-debug --disable-neon-clobber-test --disable-programs --disable-postproc --disable-doc --disable-htmlpages --disable-manpages --disable-podpages --disable-txtpages --disable-sndio --disable-schannel --disable-securetransport --disable-xlib --disable-cuda --disable-cuvid --disable-nvenc --disable-vaapi --disable-vdpau --disable-alsa --disable-cuda --disable-cuvid --disable-nvenc --disable-vaapi --disable-vdpau --enable-libx264 --enable-libxvid --enable-libx265 --enable-libvidstab --disable-sdl2 --disable-openssl --enable-zlib --enable-audiotoolbox --disable-outdev=audiotoolbox --enable-bzlib --enable-videotoolbox --enable-avfoundation --enable-iconv --disable-coreimage --disable-appkit --disable-opencl --disable-opengl --enable-gpl
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libavutil      58.  2.100 / 58.  2.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libavcodec     60.  3.100 / 60.  3.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libavformat    60.  3.100 / 60.  3.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libavdevice    60.  1.100 / 60.  1.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libavfilter     9.  3.100 /  9.  3.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libswscale      7.  1.100 /  7.  1.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 32, "message": "  libswresample   4. 10.100 /  4. 10.100
", "name": "onLog", "sessionId": 1}
 LOG  {"level": 16, "message": "file:///var/mobile/Containers/Data/Application/85F099A5-5DCB-4794-B055-53E15AD942D0/tmp/Pixel%2040%20second%20APP%20video.mp4: No such file or directory
", "name": "onLog", "sessionId": 1}
 LOG  {"errorCode": "TRIMMING_FAILED", "message": "Command failed with state Optional(\"COMPLETED\") and rc Optional(1).nil", "name": "onError"}
 LOG  {"name": "onHide"}
```

After renaming the same file to not contain spaces it trims and saves fine.

**Comment by maitrungduc1410** [comment]
hi @efstathiosntonas thanks for report the issue, it's very helpful, I didn't think about this. Will fix this soon

**Comment by maitrungduc1410** [comment]
@efstathiosntonas done, fixed in 2.2.7

**Comment by efstathiosntonas** [comment]
@maitrungduc1410, works as a charm now
